lib/components/molecules/Table/Table.style.js

```diff
@@ -0,0 +1,13 @@
+import { css } from 'styled-components';
+
+export default css`
+  border-spacing: ${props => props.theme.spacing.unit0};
```

Comment:
The variable used is wrong, UNIT_1 is the connect format and starts from 1. Please check spacing.js under theme